tools/packaging/kernel/configs/fragments/common/fs.conf

```diff
@@ -37,6 +37,8 @@ CONFIG_AUTOFS_FS=y
 CONFIG_TMPFS=y
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
+CONFIG_MISC_FILESYSTEMS=y
+CONFIG_EROFS_FS=y
 CONFIG_SIGNALFD=y
```

Comment:
You need to bump kata_config_version (https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kernel/kata_config_version) as well for the CI to pick up this config.

Reply:
Hi Amshinde,
Thanks for your review! Since the main part job of me is on the kernel side, what I'm doing now is to complete the EROFS container use cases as soon as possible (so more kernel versions can benefit this.)

Kata-container integration isn't the top priority thing compared with the new kernel features (actually many kernel TODO stuffs), I'll seek time later to investegate this.

Thanks,
Gao Xiang